packages/patternfly-4/content/get-started/getstarted-dev.md

```diff
@@ -5,8 +5,8 @@ path: "/get-started/developers"
 Before you begin, check out our [overview of PatternFly 4](/get-started/about) to get acquainted with the basic elements of the design system.
 
 Get started with:
-* [React](#react)
-* [HTML/CSS](#html/css)
+* [**React**](#react)
+* [**HTML/CSS**](#html/css)
```

Comment:
This anchor should be `(#htmlcss)`. (http://localhost:8000/get-started/developers#htmlcss)